test/unit/jobs/test_job_metrics.py

```diff
@@ -1,7 +1,27 @@
 from galaxy.job_metrics import JobMetrics
+from galaxy.job_metrics import formatting
 
 
 def test_job_metrics_load():
     # Just construct the manager to make sure all the plugin classes load fine
     # and package is configured properly.
     JobMetrics()
+
+
+def test_job_metric_formatting():
```

Comment:
If this doesn't handle singular vs. plural (second vs. second**S**), then it's enough to test for 6 cases: seconds, minutes, minutes and seconds, hours, hours and minutes, hours and minutes and seconds. Or less, if we don't display seconds if value > 60 seconds. 
However, I think it would be nice to have "1 hour and 1 minute" vs. "1 hours and 1 minutes".